misc-utils/waitpid.c

```diff
@@ -203,6 +210,10 @@ static int parse_options(int argc, char **argv)
 		}
 	}
 
+	if (allow_exited && count)
+		errx(EXIT_FAILURE,
+		     _("-e/--exited and -c/--count are incompatible"));
+
```

Comment:
It's probably overkilling for two options, but we have  `include/optutils.h` and  err_exclusive_options() to care about mutually exclusive options. We use it on many places (see `git grep ul_excl_t `).